docs/notebooks/feedback_prod.ipynb

```diff
@@ -0,0 +1,333 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<!-- docusaurus_head_meta::start\n",
+    "---\n",
+    "title: Log Feedback from Production\n",
+    "---\n",
+    "docusaurus_head_meta::end -->\n",
+    "\n",
+    "<img src=\"http://wandb.me/logo-im-png\" width=\"400\" alt=\"Weights & Biases\" />\n",
+    "<!--- @wandbcode{feedback-colab} -->"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "source": [
+    "\n",
+    "It is often hard to automatically evaluate a generated LLM response so, depending on your risk tolerance, you can gather direct user feedback to find areas to improve.\n",
+    "\n",
+    "In this tutorial, we'll use a custom RAG chatbot as an example app with which the users can interact and which allows us to collect user feedback.\n",
+    "We'll use Streamlit to build the interface and we'll capture the LLM interactions and feedback in Weave."
```

Comment:
I'm kind of confused about what's happening at a meta level here - we're using a notebook but we don't actually want the notebook to be executed, it is just for generating the markdown file?

Reply:
there's no great way to allow users to run streamlit in notebooks. Open to suggestions of how to make this possible in future.

Reply:
Maybe for the future we should extend docs generation to allow omitting the "This is a notebook / Open in Colab" header.

Instructions like `!pip install weave openai streamlit` definitely make it seem like this is intended to execute in a notebook.